src/main/java/com/example/demo/dto/CreateBookRequestDto.java

```diff
@@ -1,16 +1,29 @@
 package com.example.demo.dto;
 
+import com.example.demo.validation.Path;
+import jakarta.validation.constraints.NotNull;
+import jakarta.validation.constraints.Positive;
+import jakarta.validation.constraints.Size;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.validator.constraints.ISBN;
 
 @Data
 public class CreateBookRequestDto {
-
+    @NotNull
+    @Size(max = 25, message = "Title must be up to 25 characters")
     private String title;
+    @NotNull
+    @Size(max = 15, message = "Author name must be up to 15 characters")
```

Comment:
Is it the correct size requirements? 
For example, my full name Liliia Ponomarenko is 18 characters long. Can't I be an author of books? 🥲